examples/chat/client/chat.cc

```diff
@@ -250,29 +251,31 @@ int main(int argc, char** argv) {
     LOG(QFATAL) << "Failed to create application stub";
   }
 
-  RoomId room_id;
-  if (!absl::Base64Unescape(absl::GetFlag(FLAGS_room_id), &room_id)) {
-    LOG(QFATAL) << "Failed to parse --room_id as base 64";
-  }
-  std::unique_ptr<Room> room;
-  if (room_id.empty()) {
-    room = absl::make_unique<Room>(stub.get());
-    room_id = room->Id();
-    LOG(INFO) << "Join this room with --app_address=" << addr
-              << " --room_id=" << absl::Base64Escape(room_id);
-  }
+  if (!absl::GetFlag(FLAGS_test)) {
```

Comment:
Could you reverse the condition, so that it reads something like

```cpp
if (test) {
  return EXIT_SUCCESS;
}
```

Reply:
Done